MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -482,6 +495,10 @@ enum MSBuildSpecialCommitKind
 		NewGuid,
 		PropertyReference,
 		ItemReference,
+		Element,
```

Comment:
Element/Attribute/AttributeValue values are unneeded as the XML layer handles this, and PackageReferenceValue is unused

Reply:
removed